backend/src/controllers/uni.js

```diff
@@ -5,24 +5,28 @@ const { responseHandler } = require('../utils/helpers')
 
 /* Get all subjects */
 exports.getSubjects = function (_, res) {
+    res.header('maxAge', '365d')
     responseHandler(uniModel.getSubjects(), res)
         .catch(errorHandler(res))
 }
 
 /* Get all degrees */
 exports.getDegrees = function (_, res) {
+    res.header('maxAge', '365d')
     responseHandler(uniModel.getDegrees(), res)
         .catch(errorHandler(res))
 }
 
 /* Get all faculties */
 exports.getFaculties = function (_, res) {
+    res.header('maxAge', '365d')
     responseHandler(uniModel.getFaculties(), res)
         .catch(errorHandler(res))
 }
 
 /* Get all courses of a given subject */
 exports.getCourses = function ({ params }, res) {
+    res.header('maxAge', '365d')
```

Comment:
Might want to check this works Trav? Can you make sure the response from the API has the correct headers? Should be Cache-Control: .... in the response.